Feature/clarity engine (#75)
Zenlytic/zenlytic-docs: tips-and-tricks/zoe_context_ingestion.md

# How Zoë Ingests Context

Zoë, Zenlytic's AI data analyst, leverages multiple sources of context to understand your organization's data ecosystem, analytical philosophy, and business logic. This document explains the different ways Zoë learns about your data and how to optimize each source for better results.

## Context Sources

### 1. Custom System Prompt Context

> We allow you to extend Zoë's default system prompt with domain-specific knowledge that provides high-level organizational context:

Reviewer: ```suggestion
You can extend Zoë's default system prompt with domain-specific knowledge that provides high-level organizational context:
```
